src/java/org/apache/cassandra/db/ColumnFamilySerializer.java

```diff
@@ -71,6 +72,14 @@ public void serialize(ColumnFamily cf, DataOutputPlus out, int version)
                 written++;
             }
             assert count == written: "Table had " + count + " columns, but " + written + " written";
+            if (version >= MessagingService.VERSION_22_18)
+            {
+                out.writeBoolean(cf.isPageTokenSet());
+                if (cf.isPageTokenSet())
+                {
+                    new PageToken.Serializer(columnSerializer).serialize(cf.pageToken(), out, version);
+                }
+            }
```

Comment:
could you update the comment at the top of this method with the additions to the serialization format?